pkg/comp-functions/functions/common/instance_namespace.go

```diff
@@ -139,7 +139,7 @@ func createInstanceNamespace(serviceName, compName, claimNamespace, instanceName
 	return svc.SetDesiredKubeObjectWithName(ns, instanceNamespace, namespaceResName)
 }
 
-func createNamespacePermissions(instance string, instanceNs string, namespaceResName string, svc *runtime.ServiceRuntime) error {
+func createNamespacePermissions(instance string, instanceNs string, namespaceResName string, comp Composite, svc *runtime.ServiceRuntime) error {
```

Comment:
We could just accept the `Security` object directly here.

Reply:
True. This makes more sense.